bot/exts/utils/subscription.py

```diff
@@ -0,0 +1,148 @@
+            return True
+        return False  # user doesn't have the role
+
+    async def sub_unsub_helper(
+        self, ctx: Context, role_id: int, func: Callable, action: str
+    ) -> None:
+        """
+        Helper function for sending embeds for subscribe and unsubscribe to announcements or polls.
+
+        If `func` is `apply_role`, it checks if user have the role (returns `False`) or not (returns `True`)
+        If `func` is `remove_role`, it checks if user have the role (returns `True`) or not (returns `False`)
+        """
+        role_name = ctx.guild.get_role(role_id)
+        if await func(ctx, role_name):
+            embed = Embed(
+                title=f"{Emojis.confirmation_emoji} {action}",
+                description=f"You've {action.lower()} to {ctx.guild}'s {role_name}.",
+                color=Colours.green,
+            )
+        else:
+            embed = Embed(
+                title=f"{Emojis.warning_emoji} Already {action.lower()}",
+                description=f"You're already {action.lower()} to {ctx.guild}'s {role_name}.",
+                color=Colours.soft_red,
+            )
+        await ctx.send(content=ctx.author.mention, embed=embed)
+
+    async def sub_unsub_group_helper(
+        self, ctx: Context, func: Callable, action: str
+    ) -> None:
+        """
+        Helper function for subscribe_group and unsubscribe_group.
+
+        If `func` is `apply_role`, `role_lst` stores the role(s) which are added to the user
+        If `func` is `remove_role`, `role_lst` stores the role(s) which are removed from the user
+        """
+        roles = await self.get_roles(ctx)
+        role_lst = [role.name for role in roles if await func(ctx, role)]
+        if role_lst:
+            msg = (
+                f"{', '.join(role_lst[:-1])} and {role_lst[-1]}"
+                if len(role_lst) > 1
+                else role_lst[0]
+            )  # stores a string which tells what role(s) is/are added to or removed from the user
+            embed = Embed(
+                title=f"{Emojis.confirmation_emoji} {action}",
+                description=f"You've {action.lower()} to {ctx.guild}'s {msg}.",
+                color=Colours.green,
+            )
+        else:
+            embed = Embed(
+                title=f"{Emojis.warning_emoji} Already {action.lower()}",
+                description=f"You're already {action.lower()} to {ctx.guild}'s announcements and polls.",
+                color=Colours.soft_red,
+            )
+        await ctx.send(content=ctx.author.mention, embed=embed)
+
+    async def subscribe_list(self, ctx: Context) -> None:
+        """Sends an embed for list of Roles that you can subscribe to."""
+        roles = await self.get_roles(ctx)
+        escape = "\n- "
+        embed = Embed(
+            title=f"{Emojis.confirmation_emoji} Subscribe List",
+            description=f"- {escape.join(role.name for role in roles)}",
+            color=Colours.green,
+        )
```

Comment:
```suggestion
        embed = Embed(
            title=f"{Emojis.confirmation_emoji} Subscribe List",
            description=f"- {"\n- ".join(role.name for role in roles)}",
            color=Colours.green,
        )
```
You shouldn't need the `escape` variable here